app/views/posts/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Contents</th>
+      <th>Author</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
+      <tr>
+        <td><%= post.title %></td>
+        <td><%= post.contents %></td>
+        <td><%= User.find(post.user_id).username %></td>
```

Comment:
yeah not super proud of it. We could have a method in posControllerst:
`def author (user_id) `
`    author = User find :user_id`
`    author.username   `
` end `

and in the html
`<%= @post.author %>`